Changelog.md

```diff
@@ -10,13 +10,15 @@ Breaking
   `resolv-sync`. ([#74])
 * The trait `domain::base::octets::IntoOctets` has been merged into
   `domain::base::octets::OctetsBuilder`. It’s method `into_octets` is now
-  available as `freeze` on `OctetsBuilder. ([#75])
+  available as `freeze` on `OctetsBuilder`. ([#75])
 
 * Supprt for extended errors defined in [RFC 8914]. ([#79] by [@xofyarg])
 * New traits `domain::base::octets::OctetsFrom` and `OctetsInto` to
   convert types that are generic over octets sequences between different
   octets sequences. ([#77])
 
+* Upgrade to tokio 1.0, bytes 1.0, and latest of other dependencies ([#84])
```

Comment:
You may or may not want to name-check yourself here. So far, I’ve always did that for ‘outside contributions.’

Reply:
This sounds like a good way. I'll add myself here.